migration/migration_blackbox_test.go

```diff
@@ -411,6 +411,25 @@ func testMigration65(t *testing.T) {
 		assert.Equal(t, expectedCurrentVal, currentVal)
 	}
 }
+func testMigration66(t *testing.T) {
+	// migrate to previous version
+	migrateToVersion(sqlDB, migrations[:(initialMigratedVersion+20)], (initialMigratedVersion + 21))
+	// fill DB with data (ie, work items, links, comments, etc on different spaces)
+	assert.Nil(t, runSQLscript(sqlDB, "066-work_item_links_data_integrity.sql"))
+	// then apply the change
+	migrateToVersion(sqlDB, migrations[:(initialMigratedVersion+22)], (initialMigratedVersion + 22))
+	// verify that some records where removed
+	row := sqlDB.QueryRow("select id from work_item_links where id = '00000066-0000-0000-0000-000000000001'")
+	require.NotNil(t, row)
```

Comment:
The comment is a bit off because in the line below you're checking that a record was NOT removed.